content/docs/rfcs/16/README.md

```diff
@@ -307,20 +307,20 @@ The following structured types are defined for use on the Store API:
 
 `PagingOptions` is an `Object` containing the following fields:
 
-| Field | Type | Inclusion | Description |
-| ----: | :---: | :---: |----------- |
-| `pageSize` | `Number` | mandatory | Number of messages to retrieve per page |
-| `cursor` | [`Index`](#Index) | optional | Message [`Index`](#Index) from which to perform pagination. If not included and `forward` is set to `true`, paging will be performed from the beginning of the list. If not included and `forward` is set to `false`, paging will be performed from the end of the list.|
-| `forward` | `Bool` | mandatory | `true` if paging forward, `false` if paging backward |
+| Field |       Type        | Inclusion | Description                                                                                                                                                                                                                                                              |
```

Comment:
my editor did that automatically. it does make the md sourcecode easier to read.

Reply:
What md editor do you use? I usually edit in VSCode as I use it already for everything else, but wouldn't say it's great.

Reply:
Webstorm (JetBrains)